src/swipedeck/SwipeDeck.js

```diff
@@ -131,6 +132,10 @@ export default class SwipeDeck extends Component {
   }
 
   render() {
+    console.warn(
+      'Warning: SwipeDeck has been deprecated and will be removed in a future version of React Native Elements'
```

Comment:
Sure. Here's the source repo you can link to: https://github.com/Monte9/react-native-tinder-cards